scene/gui/popup_menu.cpp

```diff
@@ -1833,6 +1858,14 @@ bool PopupMenu::get_allow_search() const {
 	return allow_search;
 }
 
+void PopupMenu::set_max_item_height(int p_height) {
+	max_item_height = p_height;
```

Comment:
This needs to trigger some update if the value is different than the current one.